build-system/tasks/visual-diff/index.js

```diff
@@ -595,6 +598,14 @@ async function ensureOrBuildAmpRuntimeInTestMode_() {
   }
 }
 
+function installPercy_() {
+  log('info', 'Running', colors.cyan('yarn'), 'to install Percy...');
+  execOrDie('npx yarn --cwd build-system/tasks/visual-diff');
+
+  puppeteer = require('puppeteer');
```

Comment:
do we really need to be lazy with the module import?

i'd like to hoist this to the top since if ever we switch to esm (ecmascript modules), instead of cjs (commonJS, require) lazy importing/loading like this wouldn't be supported

Reply:
@erwinmombay Okay if we merge this for now, and fix later using something like https://www.npmjs.com/package/import-lazy ?

Reply:
yeah i think that would work